credentials-list-directive.js

```diff
@@ -29,6 +31,9 @@ function factory($location, brAlertService, brCredentialService, config) {
     model.state = {
       credentials: {loading: true}
     };
+    model.showHeadline = scope.showHeadline === false ? false : true;
+    model.showPublicIndicator =
+      scope.showPublicIndicator === false ? false : true;
```

Comment:
This looks like it's just simple assignment:
```js
model.showHeadline = scope.showHeadline;
model.showPublicIndicator = scope.showPublicIndicator;
```
Or if you need to coerce to booleans:
```js
model.showHeadline = !!scope.showHeadline;
model.showPublicIndicator = !!scope.showPublicIndicator;
```